commands-yml/commands/session/create.yml

```diff
@@ -18,7 +18,8 @@ example_usage:
         desiredCapabilities.setCapability(MobileCapabilityType.APP, "/path/to/ios/app.zip");
 
         URL url = new URL("http://127.0.0.1:4723");
-
+        //platform- android : windows
```

Comment:
Thanks! You can remove this line

Reply:
If I am right, I should remove the comment and replace `URL url = new URL("http://127.0.0.1:4723");` with `//URL url = new URL("http://127.0.0.1:4723/wd/hub"); ` ?

Reply:
yea, it is also good

Reply:
changes done.